framework/users.py

```diff
@@ -185,28 +189,41 @@ def is_current_user_admin(self):
 
 def get_current_user():
     if settings.UNIT_TEST_MODE:
+      user_via_env = None
       if os.environ['USER_EMAIL']!= '':
-        current_user = User(
-            email=os.environ['USER_EMAIL'],
-            _user_id=os.environ['USER_ID'])
-      else:
-        current_user = None
-      return current_user
-
-    token = session.get('id_token')
-    current_user = None
-    if token:
+        user_via_env = User(email=os.environ['USER_EMAIL'])
+      return user_via_env
+
+    # TODO(jrobbins): Remove this code path after 30 days.
+    jwt = session.get('id_token')
```

Comment:
Is this for the legacy code `session["id_token"] = token`?

Reply:
Yes.  Because, when we deploy this, we don't want existing users to be suddenly signed out.  Instead, we keep accepting their current sessions.  All those current sessions should be long gone within 30 days.